flw/src/main/java/org/motechproject/nms/flw/domain/FlwError.java

```diff
@@ -0,0 +1,62 @@
+package org.motechproject.nms.flw.domain;
+
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+
+/**
+ * Audit log object for tracking failed FLW updates
+ */
+@Entity(tableName = "nms_flw_errors")
```

Comment:
Unique for now. The goal is to eventually refactor mcts sync to go through here and create the sync error objects.... but did not want to add that into this mix